### PR TITLE
readability, minor edits

### PR DESCRIPTION
**Pull request by krisbock**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [X ] **The pull request does not introduce [breaking changes].**
- [ X] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [X ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [X ] Title of the pull request is clear and informative.
- [X ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: examples/tutorials/e2e-development/chat-with-pdf.md [hunk 1]
```diff
@@ -282,9 +287,10 @@ We have now explored how to conduct tests and evaluations for prompt flow. Addit
 
 There are several aspects we can experiment with, including but not limited to:
 
-Varying prompts for the rewrite_question and/or QnA steps.
-Adjusting the chunk size or chunk overlap during index building.
-Modifying the context limit.
+* Varying prompts for the rewrite_question and/or QnA steps.
```
Comment (zhengfeiwang) [inline examples/tutorials/e2e-development/chat-with-pdf.md:290]: any special reason to change `-` to `*` here? I used to understand they have same rendering result.

**Review by zhengfeiwang — COMMENTED** [review]
almost good to me! have some little questions, and added comment, looking forward to your response!

btw, you shall ignore those wrongly triggered workflows, we will revisit our workflow trigger rules to remove those unnecessaries.

File: examples/tutorials/e2e-development/chat-with-pdf.md (hunk 1 above)
Comment (krisbock) [reply examples/tutorials/e2e-development/chat-with-pdf.md:290]: I don't think it's resolving correctly if you look at original page "Varying prompts for the rewrite_question and/or QnA steps. Adjusting the chunk size or chunk overlap during index building. Modifying the context limit. These elements can be managed through the "config" object in the flow inputs. If you wish to experiment with the first point (varying prompts), you can add properties to the config object to control this behavior - simply by directing it to different prompt files.".

File: examples/tutorials/e2e-development/chat-with-pdf.md [hunk 2]
```diff
@@ -64,15 +65,15 @@ A "rewrite_question" step is performed before feeding the question to "find_cont
...
+Create a .env file in the second chat_with_pdf directory (same directory with the main.py) and populate it with the following content. We can use the load_dotenv() function (from the python-dotenv package) to import these into our environment variables later on. We'll delve into what these variables represent when discussing how each step of the process is implemented.
 
 Rename the .env.example file in chat_with_pdf directory and modify per your need.
 
 > If you're using Open AI, your .env should look like:
 ```ini
 OPENAI_API_KEY=<open_ai_key>
-EMBEDDING_MODEL_DEPLOYMENT_NAME=text-embedding-ada-002
-CHAT_MODEL_DEPLOYMENT_NAME=gpt-4
+EMBEDDING_MODEL_DEPLOYMENT_NAME=<text-embedding-ada-002>
```
[the remark this answers is not in the document]
Comment (krisbock) [reply examples/tutorials/e2e-development/chat-with-pdf.md:75]: yes, for clarity

**Review by zhengfeiwang — APPROVED** [review]

**Review by crazygao — APPROVED** [review]

**Pull request merged**